packages/@tinacms/graphql/src/builder/index.ts

```diff
@@ -552,6 +545,7 @@ export class Builder {
             ],
           })
         }
+      // TODO: Should we throw here?
```

Comment:
We I think we could throw an error here. It theoretically should never happen since we check to make sure the length of fields or templates is greater then 1.